src/app/qgisapp.cpp

```diff
@@ -7582,11 +7583,11 @@ void QgisApp::setLayerCRS()
 
   QgsCoordinateReferenceSystem crs( mySelector.selectedCrsId(), QgsCoordinateReferenceSystem::InternalCrsId );
 
-  Q_FOREACH ( QgsLayerTreeNode* node, mLayerTreeView->selectedNodes() )
+  Q_FOREACH( QgsLayerTreeNode* node, mLayerTreeView->selectedNodes() )
   {
     if ( QgsLayerTree::isGroup( node ) )
     {
-      Q_FOREACH ( QgsLayerTreeLayer* child, QgsLayerTree::toGroup( node )->findLayers() )
+      Q_FOREACH( QgsLayerTreeLayer* child, QgsLayerTree::toGroup( node )->findLayers() )
```

Comment:
@jef-n works fine now or at the least ```Q_FOREACH (``` has a white space. I cannot remember of having seen such a message.